src/index.css

```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
- [ ] Hi @Outkarm! The external API is to have data you can display on your pages, however, according to the design requirement:

![image](https://user-images.githubusercontent.com/77069011/221369107-72f482f6-c56b-4a37-a40a-f351658c7383.png)

According to the design, these are the possible colors and fonts:

![image](https://user-images.githubusercontent.com/77069011/221369223-bef67be4-996f-4179-bf9e-90bd1da74887.png)

You have the font right, however, the colors don't match. Please consider changing it to meet this requirement! 🤓 🔧 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/221369335-7a0335d3-7303-4929-8bcd-09f2bfbdc75d.png) | ![image](https://user-images.githubusercontent.com/79058364/233510146-1a3665cc-472a-4fcd-b63a-6bc79067157f.png)|
| ![image](https://user-images.githubusercontent.com/77069011/221369479-434dc0b4-36a9-48d7-b867-a9e1877b1968.png) |![image](https://user-images.githubusercontent.com/79058364/233510196-838af23c-c024-48de-b9ef-ae39e91ad9a9.png)



Also, If you look closely, in your home page, the elements, in your case teams have a "chess-like" color design, if the left side is light, the right side should be darker and the inverse. Please consider applying this as suggested by the project requirements! 🤓 🔧

Reply:
Resolved, 
Thanks for the comments